src/main/java/org/mule/extension/runtime/ExpirationPolicy.java

```diff
@@ -0,0 +1,35 @@
+package org.mule.extension.runtime;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A policy around when should a given instances be considered elegible for expiration.
+ * <p/>
+ * Notice that this contract is not directly tied to the instance to be expired itself. It's tied
```

Comment:
it means we do expiration based on idle time. we don't expire based on usage, creation time, etc.